client/components/disputed-order-notice/index.js

```diff
@@ -66,7 +62,7 @@ const DisputedOrderNoticeHandler = ( { chargeId, onDisableOrderRefund } ) => {
 	// This may be dead code. Leaving in for now as this is consistent with
 	// the logic before this PR.
 	// https://github.com/Automattic/woocommerce-payments/pull/7557
-	if ( dispute.status === 'lost' && ! isRefundable ) {
```

Comment:
Note that the `isRefundable` is redundant here since `isRefundable('lost')` will always be `false` if `dispute.status === 'lost'`.